working/0546-patterns/patterns-feature-specification.md

```diff
@@ -1684,30 +1677,25 @@ To type check a pattern `p` being matched against a value of type `M`:
 
         *Here, both `a` and `b` use `Object` as the matched value type.*
 
-    3.  The required type of `p` is `Map<L, W>` where:
-
-        1.  If `p` has type arguments, `L` and `W` are those type arguments.
-            *If the map pattern is explicitly typed, that wins.*
+    3.  The required type of `p` is `Map<K, V>`.
 
-        2.  Else `L` is `K` and `W` is `V`.
+* **Record**:
 
-*   **Record**:
+  1.  For each field `f` with subpattern `s` of `p`:
 
-    1.  For each field subpattern `f` of `p`:
+      1.  If `M` is a record type with the same shape as `p`, then let `F`
+          be that field's type in `M`.
 
-        1.  If `M` is a record type with a corresponding field, then let `F`
-            be that field's type.
+      2.  Else if `M` is `dynamic`, then let `F` be `dynamic`.
 
-        2.  Else if `M` is `dynamic`, then let `F` be `dynamic`.
+      3.  Else let `F` be `Object?`. *The field subpattern will only be
+          matched at runtime if the value does turn out to be a record with
+          the right shape where the field is present, so it's safe to just
+          assume the field exists when type checking here.*
 
-        3.  Else let `F` be `Object?`. *The field subpattern will only be
-            matched at runtime if the value does turn out to be a record with
-            the right shape where the field is present, so it's safe to just
-            assume the field exists when type checking here.*
+      4.  Type-check `s` using `F` as the matched value type, and find its required type.
 
-        4.  Type-check `f` using `F` as the matched value type.
-
-    3.  The required type of `p` is a record type with the same shape as `p` and
+  2.           The required type of `p` is a record type with the same shape as `p` and
```

Comment:
The spacing after "2." here is... weird.

As much as possible, I try to format the Markdown in the proposal consistently (using +4 indentation for blocks) and it might be best to try to stick with that.